mmv1/third_party/terraform/services/dataflow/resource_dataflow_flex_template_job.go.erb

```diff
@@ -585,6 +745,167 @@ func resourceDataflowFlexTemplateJobDelete(d *schema.ResourceData, meta interfac
 }
 
 func resourceDataflowFlexJobTypeCustomizeDiff(_ context.Context, d *schema.ResourceDiff, meta interface{}) error {
+
+	log.Printf("[DEBUG] I am customizing differences: machine_type: %s", "test")
```

Comment:
Probably want to clean this up!

Reply:
I thought I had gotten all of them this time 🥲